tests/compat.py

```diff
@@ -1,14 +1,12 @@
 import sys
-
+import django
 
 if sys.version_info >= (2, 7):
     import unittest
 else:
     from django.utils import unittest
 
-
-try:
-    from django.utils.encoding import force_str
-except ImportError:
-    # Required for Django < 1.5
+if django.VERSION < (1, 5):
```

Comment:
we don't support Django 1.5 anymore so we can outright remove this branch

Reply:
I have test these changes in production for my user case (just pretty maps for GeoDjango for some locations) and it works. Tests are fine and the upgrade to Django 3 was just six dependency. I do not think it will break anything. If you want, I can remove all Django 1.5 references for this pull request.